app-ios/Sources/ContributorFeature/ContributorView.swift

```diff
@@ -1,41 +1,151 @@
+                await subscriber.send(
+                    .refreshResponse(
+                        TaskResult {
+                            result
+                        }
+                    )
+                )
+            }
+        }
+        .receive(on: DispatchQueue.main.eraseToAnyScheduler())
+        .eraseToEffect()
+    case .refreshResponse(.success(let contributors)):
+        state.contributors = contributors
+        return .none
+    case .refreshResponse(.failure):
+        return .none
+    }
 }
 
 public struct ContributorView: View {
+
+    @Environment(\.openURL) var openURL
+
     private let store: Store<ContributorState, ContributorAction>
 
     public init(store: Store<ContributorState, ContributorAction>) {
         self.store = store
+
+        UITabBar.appearance().barTintColor = AssetColors.surface.color
     }
 
     public var body: some View {
-        Text("TODO: Contributor")
+        WithViewStore(store) { viewStore in
+            List(viewStore.contributors, id: \.id) { contributor in
+                ContributorItemView(contributor: contributor, action: {
+                    guard let profileUrl = contributor.profileUrl else { return }
+                    if let url = URL(string: profileUrl) {
+                        openURL(url)
+                    }
+                })
+                .listRowInsets(EdgeInsets())
+                .listRowBackground(AssetColors.background.swiftUIColor)
+                .listRowSeparator(.hidden)
+            }
+            .task {
+                await viewStore.send(.refresh).finish()
+            }
+            .listStyle(PlainListStyle())
+            .navigationTitle(L10n.Contributors.title)
+            .navigationBarTitleDisplayMode(.inline)
+        }
+    }
+}
+
+struct ContributorItemView: View {
+
+    let contributor: Contributor
+    var action: () -> Void
+
+    var body: some View {
+        ZStack {
+            HStack(spacing: 16) {
+                AsyncImage(url: URL(string: contributor.iconUrl)) { image in
+                    image.resizable()
+                } placeholder: {
+                    Color(.systemGray4)
+                }
+                .frame(width: 60, height: 60)
+                .clipShape(Circle())
+                Text(contributor.username)
+                    .font(Font.system(size: 16, weight: .bold, design: .default))
+                    .foregroundColor(AssetColors.onBackground.swiftUIColor)
+                Spacer()
+            }
+            .padding(.horizontal, 16)
+            .padding(.vertical, 12)
+
+            Button {
+                action()
+            } label: {
+                ZStack {
+                    EmptyView()
+                }
+                .frame(minWidth: 0, maxWidth: .infinity)
+                .frame(minHeight: 0, maxHeight: .infinity)
+            }
+            .buttonStyle(ListItemButtonStyle())
```

Comment:
To highlight row when selected, use ButtonStyle.

using ".listRowBackground" modifier, So that list item highlighting become not to work.